README.md

```diff
@@ -28,6 +29,20 @@ If there was no exporter configured or they haven't exported for a while it get
 
 In order to shed some more light in the dark we build a tool called zdb - Zeebe Debugger. It should help you along the way during incidents and broken systems.
 
+
+## Installation
```

Comment:
Not sure whether I would call it installation or just example usage or something. Since I would do it differently. I normally just copy it to the container, since I have it locally.

Like this:

```
[zell zdb/ cluster: zeebe-cluster ns:default]$ docker cp target/zdb-0.5.1.jar zen_moser:/usr/local/zeebe
[zell zdb/ cluster: zeebe-cluster ns:default]$ docker exec -it zen_moser bash
root@87b4a1cdbdd1:/usr/local/zeebe# java -jar 
APACHE-2.0.txt                             data/
Clarification-on-gRPC-code-generation.txt  lib/
NOTICE.txt                                 logs/
ZEEBE-COMMUNITY-LICENSE-1.0.txt            zdb
bin/                                       zdb-0.5.1.jar
config/                                    
root@87b4a1cdbdd1:/usr/local/zeebe# java -jar zdb-0.5.1.jar --version
zdb v0.5.1

```

Reply:
Proposal: Call it Example installation or Installation example

Reply:
In that case, I would rather name it `Usage`. That covers a broader meaning, and should be self-explanatory. @Zelldon what do you say?

Reply:
Sounds good

Reply:
Done.